src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1454,7 +1459,9 @@ void Partitioner::createFunction(FunctionPipeline& func_ggg) {
                 new_param_idx++;
 
                 LOG_DEBUG("Register " << prod_output << " in the function closure");
-                funcall._closure.push_back(ov::npuw::util::tensor_from_const(input_node));  // (n)/1/i/c
+                funcall._transformations.push_back(LazyTensor(
+                    TransformType::TENSOR,
+                    bank->update(std::dynamic_pointer_cast<ov::op::v0::Constant>(input_node))));  // (n)/1/i/c
```

Comment:
Shouldn't LazyTensors be managed by the Weights Bank only, so all you need to do here is to call `bank->update(input_node)` and push the returned lazy tensor to the closure?

Also, it seems your `_transformations` must be called `_lazy_closure` instead.

Reply:
Discussed + Done